### PR TITLE
psh: add df documentation

### PR DESCRIPTION
**Pull request by Maxez**

### CODE REVIEW

File: utils/psh-applets/df.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+If used with `-h` parameter it prints the help message with possible arguments and parameters as follows:
+
+```bash
+Usage: df [options] [files]
+  -T: print filesystem type
+  -i: print inode information instead of block usage
+  -h: print this help message
+```
+
+The utility prints information about the filesystem on which each file specified in `files` argument resides. The displayed information consists of:
```
Comment (agkaminski) [inline utils/psh-applets/df.md:16]:
```suggestion
The utility prints information about the filesystem on which each file specified in `files` argument resides. The information displayed consists of:
```

**Review by damianloew — COMMENTED** [review]
I've added ss, which may be helpful to show the usage of this command. Let me know what do You think. Furthermore, maybe running df on root by default could be good idea.

File: utils/psh-applets/df.md (hunk 1 above)
Comment (Maxez) [reply utils/psh-applets/df.md]: Fixed.

File: utils/psh.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ In `psh`, each command or set of commands is a separate applet, here's a list of
 
 * `bind`       - binds device to directory
 * [`cat`](psh-applets/cat.md)        - concatenate file(s) to standard output
+* [`df`](psh-applets/df.md) - print filesystem statistics
```
[the remark this answers is not in the document]
Comment (Maxez) [reply utils/psh.md]: Fixed.

**Comment by Maxez** [comment]
> I've added ss, which may be helpful to show the usage of this command. Let me know what do You think. Furthermore, maybe running df on root by default could be good idea.

Thanks for adding the example :)
The default usage of the `df` command should display all mounted filesystems info. I think that in the near future we will add mount table support to the system and fix current `df` implementation.

**Review by agkaminski — APPROVED** [review]

**Pull request merged**